extensions/2.0/Khronos/KHR_techniques_webgl/README.md

```diff
@@ -0,0 +1,838 @@
+| `MODELINVERSE`               | `FLOAT_MAT4` | Inverse of `MODEL`. |
+| `VIEWINVERSE`                | `FLOAT_MAT4` | Inverse of `VIEW`. |
+| `PROJECTIONINVERSE`          | `FLOAT_MAT4` | Inverse of `PROJECTION`. |
+| `MODELVIEWINVERSE`           | `FLOAT_MAT4` | Inverse of `MODELVIEW`. |
+| `MODELVIEWPROJECTIONINVERSE` | `FLOAT_MAT4` | Inverse of `MODELVIEWPROJECTION`. |
+| `MODELINVERSETRANSPOSE`      | `FLOAT_MAT3` | The inverse-transpose of `MODEL` without the translation.  This transforms normals in model coordinates to world coordinates. |
+| `MODELVIEWINVERSETRANSPOSE`  | `FLOAT_MAT3` | The inverse-transpose of `MODELVIEW` without the translation.  This transforms normals in model coordinates to eye coordinates. |
+| `VIEWPORT`                   | `FLOAT_VEC4` | The viewport's x, y, width, and height properties stored in the `x`, `y`, `z`, and `w` components, respectively.  For example, this is used to scale window coordinates to [0, 1]: `vec2 v = gl_FragCoord.xy / viewport.zw;` |
+| `JOINTMATRIX`                | `FLOAT_MAT4[]` | Array parameter; its length (`uniform.count`) must be greater than or equal to the length of `jointNames` array of a skin being used. Each element transforms mesh coordinates for a particular joint for skinning and animation. |
+
+For forward-compatibility, application-specific semantics must start with an underscore, e.g., `_SIMULATION_TIME`.   
+
+## Properties Reference
+
+#### JSON Schemas
+
+* [`KHR_techniques_webgl glTF extension`](#reference-khr_techniques_webgl-gltf-extension) 
+    * [`Program`](#reference-program)
+    * [`Shader`](#reference-shader)
+    * [`Technique`](#reference-technique)
+        * [`Attribute`](#reference-attribute)
+        * [`Uniform`](#reference-uniform)
+* [`KHR_techniques_webgl material extension`](#reference-khr_techniques_webgl-material-extension)
+
+---------------------------------------
+<a name="reference-khr_techniques_webgl-gltf-extension"></a>
+## KHR_techniques_webgl glTF extension
+
+Instances of shading techniques with external shader programs along with their parameterized values.  Shading techniques describe data types and semantics for GLSL vertex and fragment shader programs.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**programs**|Program `[1-*]`|An array of [`Program`](#reference-program) objects.| :white_check_mark: Yes|
+|**shaders**|Shader `[1-*]`|An array of [`Shader`](#reference-shader) objects.| :white_check_mark: Yes|
+|**techniques**|Technique `[1-*]`|An array of [`Technique`](#reference-technique) objects.| :white_check_mark: Yes|
+|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
+|**extras**|`any`|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [glTF.KHR_techniques_webgl.schema.json](schema/glTF.KHR_techniques_webgl.schema.json)
+
+### khr_techniques_webgl.gltf.extension.programs :white_check_mark: 
```

Comment:
I forgot to mention, also remove the `khr_techniques_webgl.gltf.extension.` prefixes in this section. Related to https://github.com/KhronosGroup/glTF/pull/1296#discussion_r183505232